velox/connectors/hive/tests/HivePartitionFunctionTest.cpp

```diff
@@ -277,3 +278,17 @@ TEST_F(HivePartitionFunctionTest, date) {
   assertPartitionsWithConstChannel(values, 500);
   assertPartitionsWithConstChannel(values, 997);
 }
+
+TEST_F(HivePartitionFunctionTest, HivePartitionFunctionSpecTest) {
```

Comment:
naming: test methods start with lower case and do not include 'Test'

HivePartitionFunctionSpecTest -> specToString